cmd/notation/cert.go

```diff
@@ -34,6 +34,20 @@ var (
 				Aliases: []string{"n"},
 				Usage:   "certificate name",
 			},
+			&cli.StringFlag{
+				Name:    "id",
+				Aliases: []string{"i"},
+				Usage:   "key id",
+			},
+			&cli.StringFlag{
+				Name:    "plugin",
+				Aliases: []string{"p"},
+				Usage:   "key plugin",
+			},
+			&cli.BoolFlag{
+				Name:  "kms",
+				Usage: "add key to KMS cert list",
+			},
```

Comment:
The separate list was a topic of discussion. I've got mixed feelings, but we left it like this for now. It would be great for the spec to clarify what we want.

Reply:
The current keys have `keyPath` and `certPath` properties which don't really apply to kms keys. The KMS keys only need `id` and `pluginName`.  IMO, it would be better to keep them distinct rather than building a common struct to encompass every single property across multiple key types and stash them in a single slice.